src/main/java/io/fabric8/maven/docker/config/ImageConfiguration.java

```diff
@@ -108,7 +108,7 @@ private void addVolumes(RunImageConfiguration runConfig, List<String> ret) {
 
     private void addLinks(RunImageConfiguration runConfig, List<String> ret) {
         // Custom networks can have circular links, no need to be considered for the starting order.
-        if (runConfig.getLinks() != null && !runConfig.getNetworkingConfig().isCustomNetwork()) {
+        if (!runConfig.getNetworkingConfig().isCustomNetwork()) {
```

Comment:
Could we leave this check in ? I know its not needed since `EnvUtil.splitOnLastColon()` always return a list but it makes it easier to understand what the conditions are for links to add.